### PR TITLE
Add imputes edge kind to Java analyzers

### PR DESCRIPTION
**Pull request by salguarnieri**

### CODE REVIEW

File: kythe/java/com/google/devtools/kythe/analyzers/base/EdgeKind.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
   EXPORTS("exports"),
   EXTENDS("extends"),
   GENERATES("generates"),
+  IMPUTES("imputes"),
```
Comment (schroederc) [inline kythe/java/com/google/devtools/kythe/analyzers/base/EdgeKind.java]: This should go above as one of the anchor edges.

Comment (salguarnieri) [reply kythe/java/com/google/devtools/kythe/analyzers/base/EdgeKind.java]: done

File: kythe/java/com/google/devtools/kythe/analyzers/base/EdgeKind.java [hunk 2]
```diff
@@ -23,6 +23,8 @@
   DOCUMENTS(true, "documents"),
   TAGGED(true, "tagged"),
 
+  // Edges from an anchor.
+  IMPUTES("imputes"),
```
Comment (schroederc) [inline kythe/java/com/google/devtools/kythe/analyzers/base/EdgeKind.java]: It needs to use the other constructor, that's what signifies it as an anchor edge: `IMPUTES(true, "imputes")`.

Comment (salguarnieri) [reply kythe/java/com/google/devtools/kythe/analyzers/base/EdgeKind.java]: done

**Pull request merged**